gcpy/plot/compare_zonal_mean.py

```diff
@@ -236,7 +236,10 @@ def compare_zonal_mean(
         frac_devstr = 'GCHP_dev / GCHP_ref'
 
     # If no varlist is passed, plot all 3D variables in the dataset
+    # Also remove Met_AREAM2 from GCHP StateMet collection,
+    # since we cannot make zonal mean plots from this variable.
```

Comment:
I think we can remove these comments now, right?

Reply:
@lizziel: Now fixed in eb9f72a9fb88a7c04ba80f305c35b90a400a72e5